w_titanic

```diff
@@ -0,0 +1,12 @@
+library(tidyr)
+library(dplyr)
+
+heart <- read.csv("titanic_original.csv", header = TRUE, stringsAsFactors = FALSE)
+
+heart$embarked[which(is.na(heart$embarked))] <- "S"
+heart$age[which(is.na(heart$age))] <- mean(heart$age, na.rm = TRUE)
```

Comment:
Any thoughts on question B?
"Think about other ways you could have populated the missing values in the age column. Why would you pick any of those over the mean (or not)?"